cmd/mt-gateway/api.go

```diff
@@ -21,13 +24,28 @@ type Api struct {
 //Constructs a new Api based on the passed in URLS
 func NewApi(urls Urls) Api {
 	api := Api{}
-	api.ingestHandler = withMiddleware("ingest", http.HandlerFunc(ingest.Metrics))
+	api.ingestHandler = withMiddleware("ingest", ingestHandler(urls))
 	api.graphiteHandler = withMiddleware("graphite", httputil.NewSingleHostReverseProxy(urls.graphite))
 	api.metrictankHandler = withMiddleware("metrictank", httputil.NewSingleHostReverseProxy(urls.metrictank))
 	api.bulkImportHandler = withMiddleware("bulk-importer", bulkImportHandler(urls))
 	return api
 }
 
+func ingestHandler(urls Urls) http.Handler {
+	publisher := kafka.New(strings.Split(urls.kafkaBrokers, ","), true)
+	if publisher == nil {
+		log.Info("metrics ingestion not enabled")
```

Comment:
it might be helpful to new users if the message mentions that metrics ingestion is not enabled because no kafka brokers were configured.

Reply:
87cb16c62265e5574e616489cd4489007c8f53e6